docs/contributing/setting_up_a_devcontainer.md

```diff
@@ -0,0 +1,140 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Devcontainer Setup
+
+> Step-by-Step Tutorial on How to Set Up the Repository in a Dev Container
+
+## Introduction
+
+We appreciate your interest in contributing to stdlib! Below, we've provided a step-by-step tutorial on how to set up the project locally on your device using a dev container.
+
+The stdlib repository includes a preconfigured dev container, making it the easiest way to set up your development environment. It ensures proper linting, EditorConfig, and tooling are configured right from the start.
+
+**Note:** The dev container does not yet support ARM64 architectures. For more information, or if you're interested in adding ARM64 support, you can visit this [issue][devcontainer-issue].
+
+### Prerequisites
+
+Setting up the stdlib dev container **requires** the following prerequisites:
+
+-   [Git][git]: Version control
+-   [Docker][docker]: Containerization
+-   [VS Code][vscode]: Preferred IDE
+
+### Download
+
+To acquire the source code, first navigate to the parent directory where you want to place the project’s [Git][git] repository.
+
+<!-- run-disable -->
+
+```bash
+$ cd /path/to/parent/destination/directory
+```
+
+Next, clone the repository.
+
+<!-- run-disable -->
+
+```bash
+$ git clone https://github.com/stdlib-js/stdlib.git
+```
+
+If you are wanting to contribute to stdlib, first [fork][github-fork] the repository and amend the previous command.
+
+<!-- run-disable -->
+
+```bash
+$ git clone https://github.com/<username>/stdlib.git
+```
+
+Open the repository in VS Code.
+
+```bash
+$ cd stdlib && code .
+```
+
+When prompted, open the repository in the dev container.
+
+
+<div class="image" align="center">
+    <a title="Step 1">
+        <img width="480" src="./img/step1.png" alt="Dev container setup tutorial">
```

Comment:
These alt-text description should be more descriptive. For those with screen readers, every image will read as if they are the same thing.

Reply:
See, e.g., https://accessibility.huit.harvard.edu/describe-content-images